clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/servergroup/EcsServerGroupController.java

```diff
@@ -0,0 +1,80 @@
+package com.netflix.spinnaker.clouddriver.ecs.controllers.servergroup;
+
+import com.amazonaws.services.ecs.AmazonECS;
+import com.amazonaws.services.ecs.model.DescribeServicesRequest;
+import com.amazonaws.services.ecs.model.DescribeServicesResult;
+import com.amazonaws.services.ecs.model.ServiceEvent;
+import com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider;
+import com.netflix.spinnaker.clouddriver.aws.security.NetflixAmazonCredentials;
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.ServiceCacheClient;
+import com.netflix.spinnaker.clouddriver.ecs.cache.model.Service;
+import com.netflix.spinnaker.clouddriver.ecs.model.EcsServerGroupEvent;
+import com.netflix.spinnaker.clouddriver.ecs.security.NetflixECSCredentials;
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/applications/{application}/{account}/{serverGroupName}")
```

Comment:
Would like to see this follow the conventions already in place via the aws provider... maybe more like:

```
"/applications/{application}/serverGroups/{account}/{serverGroupName}"
```

https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/controllers/AmazonClusterController.groovy

Reply:
added `serverGroups/` in the middle of `{account}/{serverGroupName}` as suggested